kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -609,12 +601,32 @@ public void createOrUpdateRchRejections(Map<String, Object> rejectedRecords, Map
         LOGGER.debug("Inserted {} and updated {} rejection records into database", createdNo, updatedNo);
     }
 
+    private void updateChildRejectionRecord(Map<String, Object> childRejects, String beneficiaryId, ChildImportRejection child, List<ChildImportRejection> updateObjects) {
+        ChildImportRejection dbChild = (ChildImportRejection) childRejects.get(beneficiaryId);
+        try {
+            Method method = dbChild.getClass().getMethod("getCreationDate");  // Get creationDate from db object and set it in the update object
+            DateTime dateTime = (DateTime) method.invoke(dbChild);
+
+            method = child.getClass().getMethod("setCreationDate", DateTime.class);
+            method.invoke(child, dateTime);
+
+            method = dbChild.getClass().getMethod("getId");
+            Long id = (Long) method.invoke(dbChild);
+            method = child.getClass().getMethod("setId", Long.class);
+            method.invoke(child, id);
+        } catch (IllegalAccessException|SecurityException|IllegalArgumentException|NoSuchMethodException|
+                InvocationTargetException e) {
+            LOGGER.error("Ignoring creation date and setting as now");
```

Comment:
Please also add argument "e" to the LOGGER.error - so that we have detailed error in log